src/domain/app/AppPageTitleRoute.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { Helmet } from 'react-helmet-async';
+import { Route, RouteProps } from 'react-router';
+import { useTranslation } from 'react-i18next';
+
+type Props = RouteProps & {
+  pageTitle: string;
+};
+
+const AppPageTitleRoute = (props: Props) => {
+  const { t } = useTranslation();
+
+  return (
+    <>
+      <Helmet>
+        <title>{t(props.pageTitle)}</title>
+        <meta property="og:title" content={t(props.pageTitle)} />
+      </Helmet>
+      <Route {...props} />
```

Comment:
Did you notice that you are passing `pageTitle` to `<Route />`?

I recommend that you use the destructuring pattern:

`({ pageTitle, ...rest })` and pass `...rest` to `<Route />`.

Reply:
Oops, I'll fix this.